programs/bpf/Cargo.lock

```diff
@@ -2081,9 +2081,8 @@ checksum = "33a33a362ce288760ec6a508b94caaec573ae7d3bbbd91b87aa0bad4456839db"
 
 [[package]]
 name = "librocksdb-sys"
-version = "0.6.1+6.28.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "81bc587013734dadb7cf23468e531aa120788b87243648be42e2d3a072186291"
+version = "0.7.1+7.3.1"
+source = "git+https://github.com/rust-rocksdb/rust-rocksdb?rev=ac4ffa3ebe5b92bff3936deac36edabc4c3f24a5#ac4ffa3ebe5b92bff3936deac36edabc4c3f24a5"
```

Comment:
Interesting that this changed, but I guess it shows up because this appears to be the only place except the top where we have a `Cargo.lock` file

Reply:
Yep, this one is a bit non-intuitive.  I noticed it only after I see a compile error.